esp/scm/ws_logaccess.ecm

```diff
@@ -246,7 +246,8 @@ ESPenum LogAccessStatusCode : int
 ESPStruct LogAccessStatus
 {
     LogAccessStatusCode Code;
-    LogAccessStatusCode Messages;
+    [min_ver("1.08")] ESParray<string> MessageArray;
+    [max_ver("1.07")] LogAccessStatusCode Messages;
```

Comment:
The type here for Messages in v1.07 should be something like a string, shouldn't it? Isn't LogAccessStatusCode an int?

Reply:
that's what we're changing in 1.08+
The reason for versioning is to keep previous interfaces around in case they're being used.
I agree it's odd, but it was recommended to version the fix.
Your legacy code should only reference 1.07, any new changes should target 1.08+
would it be easier for the ui to remove the legacy LogAccessStatsusCode Messages?

Reply:
Gotcha. So the erroneous "LogAccessStatusCode Messages" line isn't removed from the definition, but it will not show up in the runtime by default. However if there was a client that needed that field as defined would still be able to request it by targeting v.1.07